### ISSUE TITLE
Retraining pipeline

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/aframe#37]
Develop projects for building datasets for periodic retraining during first month of O3 as well as for iteratively running training (or hyperparameter searching) function on these datasets. I think the ideal structure might look something like:

- Test set injections are done once up front
- Training for the (i+1)th segment uses some subset of the ith 0-timelag test set
- Dataset of training waveforms is generated once, injected into each training set background in real-time

But open to discussion on these points. Some reasonably substantial segment of data should be held out from the retraining process altogether, so that we can optimize retraining/inference parameters on the retraining dataset then validate them on this true test set to estimate the pipeline's production performance.
